src/contracts/misc/BucketCapacityManager.sol

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {Ownable} from '@openzeppelin/contracts/access/Ownable.sol';
+import {EnumerableSet} from '@openzeppelin/contracts/utils/structs/EnumerableSet.sol';
+import {IGhoToken} from '../gho/interfaces/IGhoToken.sol';
+import {RiskCouncilControlled} from './RiskCouncilControlled.sol';
+import {IBucketCapacityManager} from './interfaces/IBucketCapacityManager.sol';
+
+/**
+ * @title BucketCapacityManager
+ * @author Aave Labs
+ * @notice Helper contract for managing bucket capacities of controlled facilitators
+ * @dev Only the Risk Council is able to action contract's functions, based on specific conditions that have been agreed upon with the community.
+ * @dev Requires GHO_TOKEN_BUCKET_MANAGER_ROLE on GhoToken
+ */
+contract BucketCapacityManager is Ownable, RiskCouncilControlled, IBucketCapacityManager {
+  using EnumerableSet for EnumerableSet.AddressSet;
+
+  /// @inheritdoc IBucketCapacityManager
+  uint256 public constant MINIMUM_DELAY = 2 days;
+
+  /// @inheritdoc IBucketCapacityManager
+  address public immutable GHO_TOKEN;
+
+  /// @inheritdoc IBucketCapacityManager
+  function RISK_COUNCIL() public view override returns (address) {
```

Comment:
Let's put view functions after write functions.
Per solidity guidelines: https://docs.soliditylang.org/en/v0.8.21/style-guide.html#order-of-functions

Reply:
Done